enterprise/frontend/src/embedding/data-picker/DataSelector/DataSelector.jsx

```diff
@@ -161,25 +156,18 @@ export class UnconnectedDataSelector extends Component {
   //
   _getComputedState(props, state) {
     const { metadata, tableFilter } = props;
-    const {
-      selectedDatabaseId,
-      selectedSchemaId,
-      selectedTableId,
-      selectedFieldId,
-    } = state;
+    const { selectedDatabaseId, selectedSchemaId, selectedTableId } = state;
 
-    let { databases, schemas, tables, fields } = props;
```

Comment:
`fields` not passed https://github.com/metabase/metabase/blob/b7e59e930ade75e768c61e52af336575d6d69489/frontend/src/metabase/querying/notebook/components/NotebookDataPicker/EmbeddingDataPicker/EmbeddingDataPicker.tsx#L78-L95